docs/netbsd.md

```diff
@@ -2,49 +2,75 @@
+```
+	* To boot up every time to the .img file:  (Have to be in the same directory as the .img file)
+```
+	 qemu-system-x86_64 -hda NetBSD.img -redir tcp:10022::22
+```
+Here the -redir flag is redirecting the 10022 on host to 22 port on guest.
+
+--->>>So far the process is tested only on linux/amd64 host. To build Go binaries do:
 ```
 make TARGETOS=netbsd
 ```
-To build C `syz-executor` binary, copy `executor/*` files to a NetBSD machine and build there with:
+
+--->>>To build C `syz-executor` binary, copy `executor/*` files to a NetBSD machine and build there with:
 ```
 gcc executor/executor_NetBSD.cc -o syz-executor -O1 -lpthread -DGOOS=\"netbsd\" -DGIT_REVISION=\"CURRENT_GIT_REVISION\"
 ```
-Then, copy out the binary back to host into `bin/netbsd_amd64` dir.
+
+"scp host@ip:/directory-of-source /directory-of-destination" command can be used to copy the files from the host to the guest.
+
+--->>>Then, copy out the binary back to host into `bin/netbsd_amd64` dir.(Inside the syzkaller directory)
 
 Building/running on a NetBSD host should work as well, but currently our `Makefile` does not work there, so you will need to do its work manually.
 
-Then, you need a NetBSD image with root ssh access with a key. General instructions can be found here [qemu instructions](https://wiki.qemu.org/Hosts/BSD).
+(Anita steps are optional if the NetBSD OS is  up and running, which it already is..)
+--->>>Then, you need a NetBSD image with root ssh access with a key. General instructions can be found here [qemu instructions](https://wiki.qemu.org/Hosts/BSD).
 
-To prepare the image, use `anita`. (You need the python module `pexpect` installed, for using Anita)
+--->>>To prepare the image, use `anita`. (You need the python module `pexpect` installed, for using Anita)
 ```
 git clone https://github.com/utkarsh009/anita
 python anita/anita --workdir anitatemp install http://nycdn.netbsd.org/pub/NetBSD-daily/netbsd-8/201710221410Z/amd64/
 ```
 NOTE: You can choose your own release tree from here: http://ftp.netbsd.org/pub/NetBSD/
 URL for a daily build might not exist in future and new release trees keep coming out.
 
-Then spin up an instance from the image generated inside `./anitatemp` directory
+--->>>Then spin up an instance from the image generated inside `./anitatemp` directory
 ```
 qemu-system-x86_64 -m 1024 -drive file=anitatemp/wd0.img,format=raw,media=disk -netdev user,id=mynet0,host=10.0.2.10,hostfwd=tcp:127.0.0.1:10022-:22 -device e1000,netdev=mynet0 -nographic
 ```
-Then create an ssh-keypair without a password and save it by the name, say, `netbsdkey`
+--->>>Then create an ssh-keypair without a password and save it by the name, say, `netbsdkey`
 
+(This is done in the host OS until specified otherwise)
 ```
 ssh-keygen -t rsa
 ```
-Then append the following to `/etc/rc.conf`
+
+(This is done in the Guest OS until specified otherwise)
+--->>>Then append the following to `/etc/rc.conf`
```

Comment:
Same here. Markdown does not treat a single new line nor --->>> as anything special.

Reply:
Yes, taken care of.